app/models/claim.rb

```diff
@@ -102,7 +102,7 @@ def data_for_calculation
   private
 
   def youth_court_fee_claimed
-    data.fetch('include_youth_court_fee_original', data['include_youth_court_fee'])
+    data['include_youth_court_fee_original'] || data['include_youth_court_fee']
```

Comment:
Be aware that if this is false, it'll check the other value. I don't think it's what you want to do.

Reply:
true (which means somewhere also this is untested - will fix this to be more specific, tbh I'm pretty sure rubocop auto-changed this from code where I check specifically if data['include_youth_court_fee_original'] is nil using .nil?

Reply:
So this is only a problem if `include_youth_court_fee_original` is false but `include_youth_court_fee` is true, i.e. if the provider didn't claim the fee but the caseworker awarded it to them anyway. _At the moment_ this can't bite us because I don't think we intend to give caseworkers functionality do that, but it would be a kindness to our future selves to do this properly just in case requirements are added.

Reply:
This is being handled in https://github.com/ministryofjustice/laa-assess-crime-forms/pull/865 (for our future selves)